test/filters/FreeTextFilter.test.js

```diff
@@ -0,0 +1,85 @@
+import { FreeTextFilter } from '../../src/filters/FreeTextFilter'
+import { describe, expect, test } from 'vitest'
+
+describe('FreeTextFilter', () => {
+  describe('matchesRecord()', () => {
```

Comment:
Still slightly disagree with your test name structure as I don't feel it reads easily. My preference would be FreeTextFilter matchesRecord _returns true when the record matches a term in the state._

We were supposed to discuss this with the team but didn't really have another standup after that. We could maybe discuss/vote on teams?